src/components/modal/v2/styles/components/_calculator.scss

```
@@ -110,6 +110,18 @@
             }
         }
 
+        &.v5Design {
+            h4 {
+                font-weight: 700;
+            }
+            label {
+                color: $v5-gray;
+            }
+            input {
+                border: 1px solid $v5-gray;
+            }
```

```suggestion
            }
            
              #purchase-amount {
                width: 100%;
            }
```
Follow up from the above.